render/jpql/src/test/kotlin/com/linecorp/kotlinjdsl/render/jpql/serializer/impl/JpqlSomeSerializerTest.kt

```diff
@@ -0,0 +1,52 @@
+package com.linecorp.kotlinjdsl.render.jpql.serializer.impl
+
+import com.linecorp.kotlinjdsl.querymodel.jpql.expression.impl.JpqlSome
+import com.linecorp.kotlinjdsl.render.TestRenderContext
+import com.linecorp.kotlinjdsl.render.jpql.serializer.JpqlRenderSerializer
+import com.linecorp.kotlinjdsl.render.jpql.serializer.JpqlSerializerTest
+import com.linecorp.kotlinjdsl.render.jpql.writer.JpqlWriter
+import io.mockk.*
+import io.mockk.impl.annotations.MockK
+import org.assertj.core.api.WithAssertions
+import org.junit.jupiter.api.Test
+
+@JpqlSerializerTest
+class JpqlSomeSerializerTest : WithAssertions {
+    private val sut = JpqlSomeSerializer()
+
+    @MockK
+    private lateinit var writer: JpqlWriter
+
+    @MockK
+    private lateinit var serializer: JpqlRenderSerializer
+
+    @Test
+    fun handledType() {
+        // when
+        val actual = sut.handledType()
+
+        // then
+        assertThat(actual).isEqualTo(JpqlSome::class)
+    }
+
+    @Test
+    fun `serialize - WHEN some is given, THEN draw some function`() {
+        // given
+        every { writer.write(any<String>()) } just runs
+        every { serializer.serialize(any(), any(), any()) } just runs
+
+        val part = mockkClass(type = JpqlSome::class, relaxed = true)
```

Comment:
I haven't written any tests for these classes, but could you tell me why you gave `relaxed = true`? Because I personally think that `relaxed` should be avoided as much as possible. 

I think `relaxed` makes me less predictable and makes it easier to miss what I really need to verify in my tests.

Reply:
`JpqlSome` does not have any factory class to make instance of it.
so, i use `mockkClass` and `relaxed = true` because test fails when it is `false`.
Is there some way to make test pass without using `relaxed` option?

Reply:
Aha, you chose this because you can only create classes like `JpqlAny` and `JpqlAll` with factory methods like `inAny()` and `inAll()` combined with `in`. 

Then I understand that you had no choice but to set `relaxed = true`. I'm in favor of this.

> And in the case of `Some`, it exists as a query model, but I'm not sure whether to open it up to the public because it's the same as `Any` in the JPQL specification. So `Some` doesn't have a factory method.